### PR TITLE
Add reproducibility video.

### PR DESCRIPTION
**Pull request by amouat**
## Type of change
Landing page for reproducibility video.

### CODE REVIEW

File: content/chainguard/chainguard-images/videos/repro.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
+---
+title: "Reproducibility and Chainguard Images"
+linktitle: "Reproducibility and Chainguard Images"
+lead: ""
+description: "This video explains the importance of reproducibility and how to reproduce any
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/repro.md:5]: Maybe we could use another word second time around so we don't say "Reproducibility/reproducible" twice? "Recreate" or "reconstruct"?

File: content/chainguard/chainguard-images/videos/repro.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+weight: 070
+toc: true
+---
+
+{{< youtube 0Qn2J89UEvI >}}
+
+## Clarification
+
+In this video we mention needing to keep copies of old APKs in order to be able to recreate images.
+This wasn't fully accuate -- in fact we do keep all our previously issued APKs, so you can build
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/repro.md:24]: " -- " > "—" (use real em dash)

File: content/chainguard/chainguard-images/videos/repro.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+  --type https://apko.dev/image-configuration \
+  --certificate-oidc-issuer https://token.actions.githubusercontent.com \
+  --certificate-identity https://github.com/chainguard-images/images/.github/workflows/release.yaml@refs/heads/main \
+  cgr.dev/chainguard/nginx:latest | jq -r .payload | base64 -d | jq .predicate > latest.apko.json
+```
+
+Building the image:
+
+```
+apk publish latest.apko.json ttl.sh/nginx-repro
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/repro.md:50]: This should be `apko`, not `apk`, right?

File: content/chainguard/chainguard-images/videos/repro.md [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+
+And it's `f705`.
+
+I've forgotten what it was before.
+
+So let's download nginx again and see if it matches.
+
+And it does.
+
+So that's pretty cool.
```
Comment (smythp) [inline content/chainguard/chainguard-images/videos/repro.md:184]: It is pretty cool.

**Review by smythp — APPROVED** [review]
Nice video, cool that the tooling lets you do this. :+1:

File: content/chainguard/chainguard-images/videos/repro.md (hunk 2 above)
Comment (amouat) [reply content/chainguard/chainguard-images/videos/repro.md:24]: Hugo actually takes care of this but I've changed it anyway.

File: content/chainguard/chainguard-images/videos/repro.md (hunk 3 above)
Comment (amouat) [reply content/chainguard/chainguard-images/videos/repro.md:50]: yeah, I wonder how that happened, it should have been a cut and paste!

File: content/chainguard/chainguard-images/videos/repro.md (hunk 1 above)
Comment (amouat) [reply content/chainguard/chainguard-images/videos/repro.md]: Changed to recreate.

**Pull request merged**